deeprank/operate/pdb.py

```diff
@@ -50,7 +50,7 @@ def get_atoms(pdb2sql):
         atom_position = numpy.array([x, y, z])
 
         # Create the atom object and link it to the residue:
```

Comment:
I realized we would also need to add a check to remove redundant contacts. 
Contact from atom A to B, and B to A are one and the same.

Reply:
I just added some extra testing for this kind of redundancy in https://github.com/DeepRank/deeprank/pull/225/commits/eaa1d29d1a1946ea36c6edcb20b529fda3126d4f. The test result was negative.

Do you have any results that indicate there are redundant contacts coming from the 'get_residue_contact_atom_pairs' function?